core/core-configuration/src/main/webapp/WEB-INF/conf/wcm-core/core-services-configuration.xml

```diff
@@ -265,6 +265,9 @@
   <component>
     <type>org.exoplatform.services.rest.DocumentRestService</type>
   </component>
+  <component>
+    <type>org.exoplatform.services.rest.DisconnectRestService</type>
```

Comment:
you can add it into org.exoplatform.ecm.connector.clouddrives package of ecms-core-connector module